module-system/sov-modules-stf-template/src/lib.rs

```diff
@@ -24,8 +24,11 @@ pub enum TxEffect {
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub enum SequencerOutcome {
+    /// Sequencer receives reward amount in defined token and can withdraw its deposit
     Rewarded(u64),
+    /// Sequencer loses its deposit and receives no reward
     Slashed(SlashingReason),
+    /// TODO: What does it actually mean? No reward, but what is with deposit?
```

Comment:
This needs to be clarified

Reply:
This means that the batch was ignored, so there is no sequencer deposit